libs/@local/hash-graph-sdk/python/scripts/codegen_blocking.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# Thanks to: https://stackoverflow.com/a/246128/9077988
+SOURCE=${BASH_SOURCE[0]}
+while [ -L "$SOURCE" ]; do # resolve $SOURCE until the file is no longer a symlink
+  DIR=$( cd -P "$( dirname "$SOURCE" )" >/dev/null 2>&1 && pwd )
+  SOURCE=$(readlink "$SOURCE")
+  [[ $SOURCE != /* ]] && SOURCE=$DIR/$SOURCE # if $SOURCE was a relative symlink, we need to resolve it relative to the path where the symlink file was located
+done
+DIR=$( cd -P "$( dirname "$SOURCE" )" >/dev/null 2>&1 && pwd )
+
+poetry run python "$DIR/generate_blocking.py"
+
+# We need to run black twice because ruff changes the output
+poetry run black "$DIR/../graph_sdk/client/blocking.py"
+poetry run ruff --fix "$DIR/../graph_sdk/client/blocking.py" || true
```

Comment:
Shouldn't `ruff` always pass or do we not guarantee that (non-blocking comment)?

Reply:
It won't pass because it complains about the line being too long, this will be fixed by the second `black` run.

```
@local/hash-graph-sdk-python:codegen:filter: graph_sdk/filter/path.py:94:89: E501 Line too long (139 > 88 characters)
@local/hash-graph-sdk-python:codegen:filter: graph_sdk/filter/path.py:98:89: E501 Line too long (123 > 88 characters)
@local/hash-graph-sdk-python:codegen:filter: graph_sdk/filter/path.py:141:89: E501 Line too long (145 > 88 characters)
@local/hash-graph-sdk-python:codegen:filter: graph_sdk/filter/path.py:153:89: E501 Line too long (112 > 88 characters)
```